pkgs/tools/graphics/mangohud/combined.nix

```diff
@@ -0,0 +1,14 @@
+{ stdenv, pkgs, lib
+, libXNVCtrl
+}:
+let
+  mangohud_64 = pkgs.callPackage ./default.nix { inherit libXNVCtrl; };
+  mangohud_32 = pkgs.pkgsi686Linux.callPackage ./default.nix { inherit libXNVCtrl; };
+in
+pkgs.buildEnv rec {
+  name = "mangohud-${mangohud_64.version}";
```

Comment:
```suggestion
  pname = "mangohud";
  version = mangohud_64.version;
```

Reply:
I don't think that's possible in `buildEnv`?

Reply:
If you rename it to pname it prints this:
```
error: anonymous function at /home/nix/nixpkgs/pkgs/build-support/buildenv/default.nix:8:2 called without required argument 'name'

       at /home/nix/nixpkgs/lib/customisation.nix:69:16:

           68|     let
           69|       result = f origArgs;
             |                ^
           70|
```

Reply:
> I don't think that's possible in buildEnv?

I did not know that.